pkgs/by-name/kv/kvazaar/package.nix

```diff
@@ -1,6 +1,7 @@
 {
   lib,
   stdenv,
+  fetchpatch,
```

Comment:
Somehow missed this in my review

Reply:
oops... I originally had this as a fetchpatch but had to switch to a substitution instead. I'll submit a PR momentarily.